ginko/src/dts/project.rs

```diff
@@ -240,12 +240,12 @@ impl Project {
     fn parse_file(&mut self, file_name: PathBuf, text: String, file_type: FileType) {
         let reader = ByteReader::from_string(text.clone());
         let lexer = Lexer::new(reader, file_name.clone().into());
-        let mut parser = Parser::new(
-            lexer,
-            ParserContext {
-                include_paths: self.include_paths.clone(),
-            },
-        );
+        // add the file's directory to the include paths to allow local includes
+        let mut include_paths = self.include_paths.clone();
+        if let Some(parent) = file_name.parent() {
+            include_paths.insert(0, parent.into());
```

Comment:
Is there a specific reason why you used `insert(0)`, and not `push`? The latter should generally be preferred unless there is a clear reason not to

Reply:
We want the path of the current file to be tried first. E.g. local files win over path files. Therefore it needs to be in position 0. Or am I missing something?